src/prepack-cli.js

```diff
@@ -48,6 +48,7 @@ function run(
     --serialize    Serializes the partially evaluated global environment as a program that recreates it. (default = true)
     --residual    Produces the residual program that results after constant folding.
     --profile    Enables console logging of profile information of different phases of prepack.
+    --statsFile  name of the output file where stats will be written to
```

Comment:
The other help messages start with capitals and end with periods. Also, it seems undesirable to abbreviate words inside the help messages.